modules/lvgl/lvgl_display.c

```diff
@@ -45,6 +45,19 @@ void lvgl_wait_cb(lv_disp_drv_t *disp_drv)
 
 #endif /* CONFIG_LV_Z_FLUSH_THREAD */
 
+#ifdef CONFIG_LV_Z_USE_ROUNDER_CB
+void lvgl_rounder_cb(lv_disp_drv_t *disp_drv, lv_area_t *area)
+{
+	__ASSERT(POPCOUNT(CONFIG_LV_Z_AREA_X_ALIGNMENT_WIDTH) == 1, "Invalid X alignment width");
+	__ASSERT(POPCOUNT(CONFIG_LV_Z_AREA_Y_ALIGNMENT_WIDTH) == 1, "Invalid Y alignment width");
```

Comment:
Does this still work as intended? The defconfig sets the X alignment, meaning `CONFIG_LV_Z_USE_ROUNDER_CB` will be `1`, so this code is built. But the `Y_ALIGNMENT_WIDTH` remains at  1, so this should error?

Reply:
You probably want to split those check into `ifs` guarding the actual calculation further down.

Reply:
This should not error as the assertion expression should still be evaluated to true when `Y_ALIGNMENT_WIDTH` is 0x1 (`POPCOUNT(0x1) == 1`). I will add `ifs` guarding to avoid unnecessary code when an axis alignment is not required.